Automattic/wp-calypso
# Hide color picker pointer

## Proposed Changes

Remove color picker pointer. I found this solution the easiest and the one with less possible side effects. 

## Testing Instructions
- Navigate to `setup/newsletterSetup?flow=newsletter`
- Open the accent color, you should not see the 'arrow down' 
<img width="575" alt="image" src="https://user-images.githubusercontent.com/2653810/193106381-1587e148-2c1d-4717-ad76-1b0d1c9d592e.png">

Related to #68353
Fixes #68353
